### ISSUE TITLE
Googleless options?

### ISSUE THREAD

**Issue by conspacer** [dimonomid/geekmarks#1]
Are there plans for non-Google authentication mechanisms for those of us who don't have a Google account?

**Comment by dimonomid** [comment]
Yeah, definitely. At least, the old good username+password authentication would be nice to have. Or, maybe auth via GitHub is more appropriate, since the primary target audience of Geekmarks is probably tech people. What's your opinion?

**Comment by ValentinTrinque** [comment]
Since most of them rely on OAuth authentication, it should be easy to integrate.

**Comment by conspacer** [comment]
Any of those other should be fine. Pure email/password is always nice for us picky purists and there are probably a horde of libraries already ready for that. 
2FA is always a nice add-on feature to grant a bit more security (again, tons of ready solutions for that).
https://github.com/search?q=TOTP

**Comment by hrmnjt** [comment]
@dimonomid are you actively maintaining geekmarks? Also, did Github login option get added to geekmarks or is this issue is still open?

**Comment by dimonomid** [comment]
I'm willing to fix bugs, but I don't have time to work on features. PRs are welcome though.
